steely/plugins/emoji.py

```diff
@@ -0,0 +1,64 @@
+import random
```

Comment:
You should add a docstring for help :sos:

Reply:
🆒